src/proto/mgmt/pool.proto

```diff
@@ -72,3 +72,19 @@ message ListPoolsResp {
 	int32 status = 1; // DAOS error code
 	repeated Pool pools = 2; // pools list
 }
+
+// ListContainers
+// Initial implementation differs from C API
+// (numContainers not provided in request - get whole list)
+message ListContReq {
+	string uuid = 1; // uuid of pool
+}
+
+message ListContResp {
+	message Cont {
+		string uuid = 1; // uuid of container
+	}
+	int32 status = 1; // DAOS error code
+	repeated Cont containers = 2; // containers
```

Comment:
why not just "repeated string containeruuids"?

Reply:
it didn't occur to me to structure it this way. Not sure if we'll end up adding more information in the container information so would prefer to keep it this way for now.